### DISCUSSION TITLE
Migrating HDs

### DISCUSSION THREAD

**Discussion by mark-wd** [vladmandic/sdnext#1380] [General]
I just bought a faster external SSD and moved the entire app folder there.  It was warning me about dubious ownership so I did a _git config --global --add safe.directory "*"_. I went to config.json and changed every occurence of the previous drive's letter to the new one. And... I am stuck and the prompt hangs forever.

```
01:11:00-479644 INFO     Verifying requirements
01:11:00-511150 INFO     Installing packages
01:11:00-598307 INFO     No changes detected: Quick launch active
01:11:00-600295 INFO     Running extension preloading
01:11:00-601783 INFO     Server arguments: ['--autolaunch']
```

What am I missing?

**Comment by Alihkhawaher** [comment]
You will need to delete the python virtual environment "venv" folder then run webui.bat. It will re-create it.

I dont know why python uses hardcoded paths in venv

**Reply by mark-wd** [reply]
You are absolutely correct. Thanks!
